Source/WebKit/UIProcess/WebAuthentication/Cocoa/WebAuthenticatorCoordinatorProxy.mm

```diff
@@ -254,6 +254,21 @@ - (void)authorizationController:(ASAuthorizationController *)controller didCompl
     return requests;
 }
 
+void WebAuthenticatorCoordinatorProxy::pauseConditionalAssertion()
+{
+    m_paused = true;
```

Comment:
I think we should early return if `m_paused` is already true?

Reply:
I agree -- we should short-circuit here if we are already paused.